src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -22,6 +22,20 @@ public enum PythonEngineVersion
         CPython3
     }
 
+    /// <summary>
+    /// EventArgs used to send the original and maigrated code to the ScriptEditor
+    /// </summary>
+    public class PythonCodeMigrationEventArgs : EventArgs
+    {
+        public string OldCode { get; private set; }
+        public string NewCode { get; private set; }
```

Comment:
might also be useful to add properties that tell us the old version and the new one so we know more about the context of the migration (2->3, 3->3.1 for example). Could be used downstream (in assistant) to launch different documentation etc based on the versions